### PR TITLE
fix #30091 ekstrabladet.dk

### LINKED ISSUE #30091: ekstrabladet.dk
### Issue URL (Ads)
[https://ekstrabladet.dk/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fekstrabladet.dk%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Mac
AdGuard version: | 1.5.14
Browser: | Chrome
Filters: | AdGuard Base,<br>AdGuard Social Media
Other extensions used: | Bitdefender

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/30091

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -487,6 +487,10 @@ alluc.ee###resultlist > div#resultitems[-ext-has="> div"]~iframe:not([src])
 winaero.com#?#aside:has(.adsbygoogle)
 moozpussy.com##article.box > blockquote:has(> a[href][target="_blank"])
 forbes.com##.article-container div:has(> div > .ntv-ad)
+ekstrabladet.dk#?#.section-area:has(> .fnSpaceManagement > div > a[href^="https://side6.dk"])
+ekstrabladet.dk#?#.fnSpaceManagement:has(> div > div > div > .drfront > .df-articles > .df-container > .df-container-inner > .df-article > .df-article-content > .df-img-container > div > a[href^="https://shero.dk"])
+ekstrabladet.dk#?#.fnSpaceManagement:has(> .line > ul > li > a[href^="https://shero.dk"])
```
Comment (BlazDT) [inline EnglishFilter/sections/css_extended.txt]: Not shown in code for me. Also please move all rules to foreign.

File: EnglishFilter/sections/foreign.txt [hunk 2]
```diff
@@ -1774,7 +1774,8 @@ newz.dk##.top-interstitial
 information.dk##.ib-wrap
 @@||information.dk/adframe.js
 ! https://github.com/AdguardTeam/AdguardFilters/issues/12132
-ekstrabladet.dk##.article-wrapper > .grid-column--xl > .section-area--article > .eb-row > .eb-row-item[style="width: 66%"]
+! https://github.com/AdguardTeam/AdguardFilters/issues/30091
+ekstrabladet.dk##.section-area--standard > .eb-row > .eb-row-item[style="width: 25%;"]
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: Not shown in code.

**Review by BlazDT — CHANGES_REQUESTED** [review]

**Review by BlazDT — APPROVED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 2 above)
Comment (slavaleleka) [reply EnglishFilter/sections/foreign.txt]: I can't find it too

**Review by slavaleleka — COMMENTED** [review]
We probably should also block items under [an article, for example](https://ekstrabladet.dk/sport/fodbold/dansk_fodbold/superligaen/fc_nordsjaelland/superliga-profil-solgt-til-tyskland/7690026), which are redirecting to third-party websites  
`ekstrabladet.dk##.plistaList > a:not([data-lpurl^="https://ekstrabladet.dk"])`

File: EnglishFilter/sections/foreign.txt (hunk 2 above)
Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: Here
<details><summary>Screenshot:</summary>

Снимок экрана 2019-06-27 в 13 33 10

</details><br/>

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/css_extended.txt]: 1, 2, 3 - not ads

File: EnglishFilter/sections/foreign.txt (hunk 2 above)
Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt]: it is not ad

**Review by Alex-302 — COMMENTED** [review]
not sure if we should to block these links

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**